### PR TITLE
Bump Android Support Library to 27.1.1

### PR DESCRIPTION
**Pull request by dulmandakh**
This PR bumps Android Support Library version to 27.1.1.

FYI, originally was a part of https://github.com/facebook/react-native/pull/20026.

Test Plan:
----------
Everything builds and runs as normal.

Release Notes:
--------------

[ANDROID] [ENHANCEMENT] [Android Support Library] - Bump to version 27.1.1

### CODE REVIEW

File: local-cli/templates/HelloWorld/android/build.gradle [hunk 1]
```diff
@@ -36,10 +36,10 @@ ext {
     minSdkVersion = 16
     compileSdkVersion = 26
     targetSdkVersion = 26
-    supportLibVersion = "26.1.0"
+    supportLibVersion = "27.1.1"
 }
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '4.4'
+    gradleVersion = '4.8'
```
Comment (gengjiawen) [inline local-cli/templates/HelloWorld/android/build.gradle]: why 4.8 here ?

Comment (gengjiawen) [reply local-cli/templates/HelloWorld/android/build.gradle]: also use wrapper task to update, don't change gradle-wrapper.properties directly.

**Comment by dulmandakh** [comment]
@facebook-github-bot shipit

**Comment by gengjiawen** [comment]
maybe wait for 28 stable release.

**Comment by dulmandakh** [comment]
@gengjiawen maybe, but it might be better to migrate to AndroidX once it becomes stable.

**Comment by dulmandakh** [comment]
@hramos please review and merge. I found that some apps and libraries from Android use support library 27.x version.

**Comment by gengjiawen** [comment]
I think the most import thing is android gradle plugin compatible to 3.x, can we put that first ? @dulmandakh .

**Comment by gengjiawen** [comment]
this pr is good to go. This pr is split from a big pr, I want android gradle plugin compatible to 3.x  comes first since it's more important. Bump to 28 can wait since it comes out recently.

**Comment by dulmandakh** [comment]
rebased to master, and it works. please merge 👍

**Comment by dulmandakh** [comment]
@gengjiawen we shouldn't be blocking unrelated PRs because of gradle plugin 3.x. Or we should be working to improve RN tooling where possible. 

Yes, I agree that gradle plugin 3.x support is very important, so I took https://gradle.org/training/intro-to-gradle/ training and learnt some basics about gradle. Also I'm planning to create small PRs to cleanup and refactor gradle scripts, then make it compatible with gradle plugin 3.x.

**Comment by gengjiawen** [comment]
For now, the script is compatible with gradle 3.x, I think we should do it now.

**Pull request closed without merging**